devito/at_controller.py

```diff
@@ -0,0 +1,344 @@
+import os
+import stat
+import math
+import shutil
+import cpuinfo
+import subprocess
+
+# global vars.
+# report folder will be created in current dir. Change if necessary
+reports_folder_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "auto-tune-reports")
```

Comment:
Because retrieving block size from report isint part of whole auto tuning process. Thus I did not want to want to initialise an object (which would raise exception if there is no isat installed) for no reason. Lets say users reuse report copy from other machine. In any case Im gonna set is as env var for now, until config/env question is resolved